src/library/library.cpp

```diff
@@ -369,6 +455,94 @@ QStringList Library::getDirs() {
     return m_pTrackCollection->getDirectoryDAO().getDirs();
 }
 
+void Library::paneCollapsed(int paneId) {
```

Comment:
I guess we don't need the collapse related code in this two-pane PR.... Shall we pull it out of the PR and put it back in later when we need it? Or is it too much work?